lib/storage/client.js

```diff
@@ -1182,6 +1182,85 @@ class Client extends CryptoConsumer {
     })
     return listGroups
   }
-}
 
+  /**
+   * addGroupMembers adds members to the TozStore Group given.
+   *
+   * @param {string} groupId  UUID assigned by TozStore, used to identify a group.
+   * @param {GroupMember[]} groupMembers wraps together the ClientID, Capability and Membership Key for each client being added to the group
+   *
+   *
+   * @returns {GroupMember} A response from TozStore; the group members who have been added.
+   *
+   */
+  async addGroupMembers(groupId, groupMembers = []) {
+    // Get Group Membership key for current client
+    let membershipResponse = await this.authenticator.tsv1Fetch(
+      `${this.config.apiUrl}/v2/storage/groups/${groupId}/membership_key`,
+      {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      }
+    )
+    let groupMembershipJson = await validateResponseAsJSON(membershipResponse)
+    // Get All members Public Keys to create their membership keys
+    let publicKeysResponse = await this.authenticator.tokenFetch(
+      `${this.config.apiUrl}/v1/client/${groupMembers.clientID}/public`,
```

Comment:
This method will likely fail when done in batch -- The batch call is a POST endpoint living at /v1/client/public and it accepts:
```
type batchPublicRequest struct {
	ClientIDs []uuid.UUID `json:"client_ids"`
}
```

Reply:
i couldnt get the array to work so i opted to do this to atleast get some functionality working!, but fixing that now